main.py

```diff
@@ -48,13 +51,14 @@ def get(location: str, appointment_type: str, num_people: str, date: str) -> str
             'There is no appointment slots at all. It is very suspicious. But it can be a temporary problem',
             category=ExternalResourceHasChanged,
         )
+        return ''
     earliest_appointment_info = js[0]
     try:
         earliest_date = earliest_appointment_info['date']
     except KeyError:
         raise ExternalResourceHasChanged('The IND resource has changed the appointment scheme.')
 
-    if earliest_date < date:
+    if dt.datetime.strptime(earliest_date, '%Y-%m-%d') < dt.datetime.strptime(date, '%d-%m-%Y'):
```

Comment:
That's right point. I do not know why the initial author decided to compare dates in this format as strings.